src/indexing.jl

```diff
@@ -243,7 +243,7 @@ indices calling [`to_axis`](@ref).
 @inline function to_axes(A, inds::Tuple)
     if ndims(A) === 1
         return (to_axis(axes(A, 1), first(inds)),)
-    elseif length(inds) === 1
+    elseif dynamic(length(inds)) === 1
```

Comment:
You can just do `Base.length(inds)` here to avoid having to explicitly convert to `Int`.